modules/atom/pom.xml

```diff
@@ -206,7 +206,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.activemq</groupId>
-            <artifactId>activemq-core</artifactId>
+            <artifactId>activemq-broker</artifactId>
```

Comment:
I understand this is forced by activemq, but I think that a migration guide entry saying so and warning the user that he has to look into this reduces risk on our side. Also, AFAIK, using -all is not a recommendation we give (might be wrong though)